### PR TITLE
[DD] Keep the network drivers infs 8.3 like on MS Windows

### PR DESCRIPTION
**Pull request by JoachimHenze**
Windows has all its network drivers inf files in 8.3. We were pretty close to that too in the past.

This is an addendum to the "diabolic" 0.4.15-dev-52-g 019f21ee1d764dff8586acc847a299714e83977a which moved us further away from our target.

The commit on its own was a good idea actually to get the inf into the drivers directory and therefore simplifies adding and excluding drivers more locally (e.g. excluding works by adding a single hashtag within the specific drivers CMakeLists.txt now).

But that cool feature doesn't require us to move away from our target.

For the record:
The historic netrtpnt.inf and netisa.inf (both 8.3) have been merged to netne2000.inf (not 8.3) by 0.4.15-dev-2763-g 6924b8ff3985224a2d094ce509e4b843f387ab32 Unification of that was cool too,
but again: 8.3 is superior. Therefore netne.inf for that.

Worst of all: Since the .inf files did contain their filename within a comment also, that means the historic renaming by the mentioned commit invalidated that comment.
netamd.inf fixed by restoring the ancient filename before 0.4.15-dev-52-g 019f21ee1d764dff8586acc847a299714e83977a
netrtl.inf fixed by restoring the ancient filename before 0.4.15-dev-52-g 019f21ee1d764dff8586acc847a299714e83977a
netne.inf I fixed the comment manually.

**All network drivers infs are 8.3 afterwards.**

### CODE REVIEW

**Review by DarkFire01 — CHANGES_REQUESTED** [review]
No.

they made way more sense before,
None of these drivers encompasses all the devices the name you’ve changed it to implies

**Comment by JoachimHenze** [comment]
> they made way more sense before,

Yes, I do agree, they made a lot more sense in releases/0.4.14 and before.
Also currently the names in the comments of the drivers infs are wrong.

**Comment by DarkFire01** [comment]
> > they made way more sense before,
> 
> Yes, I do agree, they made a lot more sense in releases/0.4.14 and before. Also currently the names in the comments of the drivers infs are wrong. **We do what Windows does!**

Not anymore.
Stop copying stupid stuff from windows to copy stupid stuff from windows.

**Comment by cbialorucki** [comment]
> This is an addendum to the diabolic 0.4.15-dev-52-g https://github.com/reactos/reactos/commit/019f21ee1d764dff8586acc847a299714e83977a which moved us further away from our target.

> that cool feature doesn't require us to move away from our target.

We don't target Windows Server 2003 anymore. Feel free to run your own fork that does.

> 8.3 is superior. Therefore netne.inf for that.

Why is 8.3 superior? I understand wanting to use three letter file extensions generally, but we're not DOS anymore. Is it really worth disturbing the hornet's nest?

**Comment by JoachimHenze** [comment]
It is closer to what Windows ships out of the box, and it also helps to keep in sync with the older releases.
There is no benefit in making those file names longer than necessary.

**Comment by oleg-dubinskiy** [comment]
Btw, by the same manner, we can also rename some other drivers including the ones unrelated to networking, to match Windows XP/2003 (and to make their replacement easier): ac97 -> ac97intc (since the driver actually works only with Intel audio controllers), e1000 -> e1000325 (for Desktop network card version) or e1000nt5 (for Server network card version) etc etc... (**in case we really want to do that!**)

**Review by binarymaster — APPROVED** [review]
No strong opinion on this, as this wouldn't hurt my feelings in any way.

**Review by HBelusca — COMMENTED** [review]
image
and (searching with `netne* netp* netam* netrt*` in WinRAR):
image
image

In conclusion, I'm not convinced of the pertinence of this PR.

**Comment by HBelusca** [comment]
> [...] e1000 -> e1000325 (for Desktop network card version) or e1000nt5 (for Server network card version) etc etc...

OK that example would generate a messier situation than what's already present.

File: drivers/network/dd/ne2000/CMakeLists.txt [hunk 1]
```diff
@@ -11,4 +11,4 @@ add_pch(ne2000 include/ne2000.h SOURCE)
 set_module_type(ne2000 kernelmodedriver)
 add_importlibs(ne2000 ndis ntoskrnl hal)
 add_cd_file(TARGET ne2000 DESTINATION reactos/system32/drivers FOR all)
-add_driver_inf(ne2000 netne2000.inf)
+add_driver_inf(ne2000 netne.inf)
```
Comment (SergeGautherie) [inline drivers/network/dd/ne2000/CMakeLists.txt:14]: Note:
This one could be renamed `netne2k.inf`, if ever wanted.

Comment (JoachimHenze) [reply drivers/network/dd/ne2000/CMakeLists.txt:14]: Yes, that would be possible as well. I do agree.

Comment (binarymaster) [reply drivers/network/dd/ne2000/CMakeLists.txt:14]: Yeah, Novell NE1000 exists, though it's an old ISA card.

**Pull request merged**